docs/devops.md

```diff
@@ -450,56 +450,57 @@ sudo apt install build-essential
 
 Provisioning VMs with the Code
 
-1. Install Node LTS.
-
-2. Update `npm` and install PM2 and setup `logrotate` and startup on boot
+1. Install `pnpm` globally.
 
    ```console
-   npm i -g npm@8
-   npm i -g pm2
-   pm2 install pm2-logrotate
-   pm2 startup
+   curl -fsSL https://get.pnpm.io/install.sh | sh -
    ```
 
-3. Clone freeCodeCamp, setup env and keys.
+2. Clone freeCodeCamp, setup env and keys.
 
    ```console
    git clone https://github.com/freeCodeCamp/freeCodeCamp.git
    cd freeCodeCamp
    git checkout prod-current # or any other branch to be deployed
    ```
 
-4. Create the `.env` from the secure credentials storage.
+3. Create the `.env` from the secure credentials storage.
 
-5. Create the `google-credentials.json` from the secure credentials storage.
+4. Create the `google-credentials.json` from the secure credentials storage.
 
-6. Install dependencies
+5. Install dependencies
 
    ```console
    pnpm install
    ```
 
+6. Setup pm2 `logrotate` and startup on boot
+  
+   ```console
+   pnpm pm2 install pm2-logrotate
+   pnpm pm2 startup
+   ```
+
 7. Build the server
 
    ```console
-   pnpm run prebuild && pnpm run build:curriculum && pnpm run build:server
+   pnpm prebuild && pnpm build:curriculum && pnpm build:server
```

Comment:
Why does this not have the `run`, but the one below does?

Reply:
I forgot to update the other one, I'll do that in a followup PR.